src/pages-and-resources/discussions/data/api.js

```diff
@@ -61,6 +61,7 @@ function normalizePluginConfig(data) {
   return {
     allowAnonymousPosts: data.allow_anonymous,
     allowAnonymousPostsPeers: data.allow_anonymous_to_peers,
+    reportedEmailNotifications: data.reported_email_notifications,
```

Comment:
@awais-ansari can this be renamed to `data.reported_content_email_notification`?

Reply:
Sure